### PR TITLE
Mid-May pull

### LINKED ISSUE #1581: new list -> stamparm malware domain
list ->
https://raw.githubusercontent.com/stamparm/aux/master/maltrail-malware-domains.txt

### LINKED ISSUE #1582: new list -> cybersaian italian
Source ->
https://github.com/CyberSaiyanIT/InfoSharing

list (domain) ->
https://infosharing.cybersaiyan.it/feeds/CS-PIHOLE

### PR DESCRIPTION
**Pull request by DandelionSprout**
* Added 15 new lists and 7 new maintainers.
* Added a few list mirrors.
* Added donation links for SecLists.
* Fixes #1581.
* Fixes #1582.

### CODE REVIEW

File: data/FilterListMaintainer.json [hunk 1]
```diff
@@ -4986,5 +5018,201 @@
...
+  {
+    "filterListId": 2049,
+    "maintainerId": 152
+  },
+  {
+    "filterListId": 2050,
+    "maintainerId": 152
+  },
+  {
+    "filterListId": ,
```
Comment (collinbarrett) [inline data/FilterListMaintainer.json:5131]: missing `filterlistId` here is breaking the CI

Comment (DandelionSprout) [reply data/FilterListMaintainer.json]: Thanks. Seems to have been fixed now.

**Comment by DandelionSprout** [comment]
Seems to me like the newest error is not my fault this time around, at least.

File: data/FilterListTag.json [hunk 2]
```diff
@@ -11014,5 +11014,69 @@
   {
     "filterListId": 2209,
     "tagId": 38
+  },
+  {
+    "filterListId": 2115,
```
Comment (collinbarrett) [inline data/FilterListTag.json:11019]:
```suggestion
    "filterListId": 2215,
```

File: data/FilterListTag.json [hunk 3]
```diff
@@ -11014,5 +11014,69 @@
...
+    "tagId": 6
+  },
+  {
+    "filterListId": 2115,
```
Comment (collinbarrett) [inline data/FilterListTag.json:11023]:
```suggestion
    "filterListId": 2215,
```

File: data/FilterListTag.json [hunk 4]
```diff
@@ -11014,5 +11014,69 @@
...
+    "tagId": 7
+  },
+  {
+    "filterListId": 2116,
```
Comment (collinbarrett) [inline data/FilterListTag.json:11027]:
```suggestion
    "filterListId": 2216,
```

File: data/FilterListTag.json [hunk 5]
```diff
@@ -11014,5 +11014,69 @@
...
+    "tagId": 6
+  },
+  {
+    "filterListId": 2117,
```
Comment (collinbarrett) [inline data/FilterListTag.json:11031]:
```suggestion
    "filterListId": 2217,
```

File: data/FilterListTag.json [hunk 6]
```diff
@@ -11014,5 +11014,69 @@
...
+    "tagId": 6
+  },
+  {
+    "filterListId": 2118,
```
Comment (collinbarrett) [inline data/FilterListTag.json:11035]:
```suggestion
    "filterListId": 2218,
```

File: data/FilterListTag.json [hunk 7]
```diff
@@ -11014,5 +11014,69 @@
...
+    "tagId": 6
+  },
+  {
+    "filterListId": 2119,
```
Comment (collinbarrett) [inline data/FilterListTag.json:11039]:
```suggestion
    "filterListId": 2219,
```

File: data/FilterListTag.json [hunk 8]
```diff
@@ -11014,5 +11014,69 @@
...
+    "tagId": 6
+  },
+  {
+    "filterListId": 2120,
```
Comment (collinbarrett) [inline data/FilterListTag.json:11043]:
```suggestion
    "filterListId": 2220,
```

File: data/FilterListTag.json [hunk 9]
```diff
@@ -11014,5 +11014,69 @@
...
+    "tagId": 30
+  },
+  {
+    "filterListId": 2121,
```
Comment (collinbarrett) [inline data/FilterListTag.json:11047]:
```suggestion
    "filterListId": 2221,
```

File: data/FilterListTag.json [hunk 10]
```diff
@@ -11014,5 +11014,69 @@
...
+    "tagId": 3
+  },
+  {
+    "filterListId": 2122,
```
Comment (collinbarrett) [inline data/FilterListTag.json:11051]:
```suggestion
    "filterListId": 2222,
```

File: data/FilterListTag.json [hunk 11]
```diff
@@ -11014,5 +11014,69 @@
...
+    "tagId": 3
+  },
+  {
+    "filterListId": 2123,
```
Comment (collinbarrett) [inline data/FilterListTag.json:11055]:
```suggestion
    "filterListId": 2223,
```

File: data/FilterListTag.json [hunk 12]
```diff
@@ -11014,5 +11014,69 @@
...
+    "tagId": 3
+  },
+  {
+    "filterListId": 2124,
```
Comment (collinbarrett) [inline data/FilterListTag.json:11059]:
```suggestion
    "filterListId": 2224,
```

File: data/FilterListTag.json [hunk 13]
```diff
@@ -11014,5 +11014,69 @@
...
+    "tagId": 3
+  },
+  {
+    "filterListId": 2125,
```
Comment (collinbarrett) [inline data/FilterListTag.json:11063]:
```suggestion
    "filterListId": 2225,
```

File: data/FilterListTag.json [hunk 14]
```diff
@@ -11014,5 +11014,69 @@
...
+    "tagId": 3
+  },
+  {
+    "filterListId": 2126,
```
Comment (collinbarrett) [inline data/FilterListTag.json:11067]:
```suggestion
    "filterListId": 2226,
```

File: data/FilterListTag.json [hunk 15]
```diff
@@ -11014,5 +11014,69 @@
...
+    "tagId": 7
+  },
+  {
+    "filterListId": 2127,
```
Comment (collinbarrett) [inline data/FilterListTag.json:11071]:
```suggestion
    "filterListId": 2227,
```

File: data/FilterListTag.json [hunk 16]
```diff
@@ -11014,5 +11014,69 @@
...
+    "tagId": 7
+  },
+  {
+    "filterListId": 2128,
```
Comment (collinbarrett) [inline data/FilterListTag.json:11075]:
```suggestion
    "filterListId": 2228,
```

File: data/FilterListTag.json [hunk 17]
```diff
@@ -11014,5 +11014,69 @@
...
+    "tagId": 7
+  },
+  {
+    "filterListId": 2129,
```
Comment (collinbarrett) [inline data/FilterListTag.json:11079]:
```suggestion
    "filterListId": 2229,
```

**Pull request merged**